src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = {};
+  const changeCss = sourceString.split(';');
+
+  changeCss.forEach((item) => {
+    const [key, value] = item.split(':');
```

Comment:
The `split(':')` operation assumes that there is exactly one colon per item. Consider handling cases where there might be more than one colon or no colon at all, which could lead to unexpected results.